mw-webapp/src/logic/waysTable/OwnWaysTable.tsx

```diff
@@ -0,0 +1,47 @@
+import {useEffect, useState} from "react";
+import {WayPreviewDAL} from "src/dataAccessLogic/WayPreviewDAL";
+import {columns} from "src/logic/waysTable/columns";
+import {WaysTable} from "src/logic/waysTable/WaysTable";
+import {WayPreview} from "src/model/businessModelPreview/WayPreview";
+
+export type OwnWaysTableProps = {
```

Comment:
Maybe I'm wrong but usually we describe props by interface not type?

Reply:
and wht we need export here? Do you use this props in other files too?

Reply:
My bad, fixed